views/layout.hbs

```diff
@@ -11,6 +11,16 @@
 
   <body>
 
```

Comment:
Here! You can add a condition on the handlebars to show different links, for example - if the user exists you will see links to logout and private page, else signup, login and main page.